library/Recurly/V3/API/Account.hs

```diff
@@ -28,6 +28,15 @@ getAccountBillingInfo accountCode = Recurly.liftRecurly $ do
     ["accounts", into @PathPiece.PathPiece accountCode, "billing_info"]
   RecurlyApi.sendRequest request
 
+getAccountBillingInfos
+  :: Recurly.MonadRecurly m
+  => Account.Code
+  -> m (Client.Response (Either RecurlyException [BillingInfo]))
```

Comment:
As far as I can tell, this endpoint only ever returns one top-level object: https://developers.recurly.com/api/v2021-02-25/index.html#operation/get_billing_info

Reply:
Hm I believe you're right, I'll close this for now while trying to noodle on how to allow for multiple billings.